### PR TITLE
Updates for BLAST+ 2.15.0

### PR DESCRIPTION
**Pull request by christiam**

### CODE REVIEW

File: blast/2.15.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+USER root
+WORKDIR /root/
+RUN mkdir -p /blast/bin /blast/lib
+
+RUN apt-get -y -m update && apt-get install -y build-essential curl libidn12 libnet-perl perl-doc liblmdb-dev wget libsqlite3-dev \
+    libgomp1 libxml-simple-perl libjson-perl parallel vmtouch cpanminus && \
+    rm -fr /var/lib/apt/lists/*  && \
+    cpanm HTML::Entities
+
+# cmake installed by apt is currently too old
```
Comment (boratyng) [inline blast/2.15.0/Dockerfile:17]:
```suggestion
```

File: blast/2.15.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+RUN wget https://github.com/Kitware/CMake/releases/download/v3.22.3/cmake-3.22.3-linux-x86_64.tar.gz && tar xfz cmake-3.22.3-linux-x86_64.tar.gz && rm cmake-3.22.3-linux-x86_64.tar.gz
```
Comment (boratyng) [inline blast/2.15.0/Dockerfile:18]:
```suggestion
```
This is not needed if `cmake` is installed with `apt-get`.

File: blast/2.15.0/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+ENV PATH="${PATH}:/root/cmake-3.22.3-linux-x86_64/bin"
```
Comment (boratyng) [inline blast/2.15.0/Dockerfile:19]:
```suggestion
```

File: blast/2.15.0/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+ARG blast_version
+ARG vdb_version
+ARG num_procs=8
+LABEL Description="NCBI BLAST" Vendor="NCBI/NLM/NIH" Version=${blast_version} Maintainer=camacho@ncbi.nlm.nih.gov
+
+USER root
+WORKDIR /root/
+RUN mkdir -p /blast/bin /blast/lib
+
+RUN apt-get -y -m update && apt-get install -y build-essential curl libidn12 libnet-perl perl-doc liblmdb-dev wget libsqlite3-dev \
```
Comment (boratyng) [inline blast/2.15.0/Dockerfile:12]:
```suggestion
RUN apt-get -y -m update && apt-get install -y build-essential curl libidn12 libnet-perl perl-doc liblmdb-dev wget libsqlite3-dev cmake \
```
In ubuntu:22.04 you can use `cmake` installed by `apt-get`.

**Review by boratyng — CHANGES_REQUESTED** [review]

**Comment by christiam** [comment]
Thanks for the feedback @boratyng  - I incorporated your suggestions in 250317e

**Review by boratyng — APPROVED** [review]

**Pull request merged**